lib/gitgitgadget.ts

```diff
@@ -260,14 +260,19 @@ export class GitGitGadget {
         // update work repo from base
         await this.updateNotesAndPullRef(pr.number, previousTag);
 
-        // Remove template from description
-        let prTemplate =
-            await git(["show",
-                       "upstream/master:.github/PULL_REQUEST_TEMPLATE.md"],
-                      { workDir: this.workDir });
-        // github uses \r\n so make sure it is set
-        prTemplate = prTemplate.replace(/([^\r])\n/, "$1\r\n");
-        const prBody = pr.body.replace(prTemplate, "");
+        // Remove template from description (if template exists)
+        let prBody: string;
+        try {
+            let prTemplate =
+                await git(["show",
+                           "upstream/master:.github/PULL_REQUEST_TEMPLATE.md"],
+                          { workDir: this.workDir });
+            // github uses \r\n so make sure it is set
+            prTemplate = prTemplate.replace(/([^\r])\n/, "$1\r\n");
```

Comment:
Oh, and wouldn't this be better `.replace(/\r?\n/g, "\r\n")`? I.e. replace _all_ occurrences, and look for an optional `\r` before the `\n`?

Reply:
I had intended to do a global replace.  The regex excludes `\r` but I like yours better.  Pushed an update that only changes the regex.

Reply:
(Technically, it is probably poor practice to sneak in that fix in a commit that claims to only surround that code block in `try`/`catch`... but I won't object if you do not want to separate those two changes into two commits.)